simple-page-ordering.php

```diff
@@ -103,18 +103,39 @@ public static function wp() {
 			$orderby = get_query_var( 'orderby' );
 			$screen  = get_current_screen();
 			if ( ( is_string( $orderby ) && 0 === strpos( $orderby, 'menu_order' ) ) || ( isset( $orderby['menu_order'] ) && 'ASC' === $orderby['menu_order'] ) ) {
-				$script_name = ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) ? '/assets/js/src/simple-page-ordering.js' : '/assets/js/simple-page-ordering.min.js';
-				wp_enqueue_script( 'simple-page-ordering', plugins_url( $script_name, __FILE__ ), array( 'jquery-ui-sortable' ), '2.1', true );
-				wp_localize_script(
-					'simple-page-ordering',
-					'simple_page_ordering_localized_data',
-					array(
-						'_wpnonce'  => wp_create_nonce( 'simple-page-ordering_' . $screen->id ),
-						'screen_id' => (string) $screen->id,
-					)
-				);
 
-				wp_enqueue_style( 'simple-page-ordering', plugins_url( '/assets/css/simple-page-ordering.css', __FILE__ ), [], SIMPLE_PAGE_ORDERING_VERSION );
+				$script_name       = 'dist/js/simple-page-ordering.js';
+				$script_asset_path = plugin_dir_path( __FILE__ ) . 'dist/js/simple-page-ordering.asset.php';
+				$script_asset      = file_exists( $script_asset_path )
+					? require $script_asset_path
+					: false;
+
+				if ( false !== $script_asset ) {
+					$script_url = plugins_url( $script_name, __FILE__ );
+					wp_enqueue_script( 'simple-page-ordering', $script_url, $script_asset['dependencies'], $script_asset['version'], true );
+
+					wp_localize_script(
+						'simple-page-ordering',
+						'simple_page_ordering_localized_data',
+						array(
+							'_wpnonce'  => wp_create_nonce( 'simple-page-ordering_' . $screen->id ),
+							'screen_id' => (string) $screen->id,
+						)
+					);
+
+					wp_enqueue_style( 'simple-page-ordering', plugins_url( '/dist/css/simple-page-ordering.css', __FILE__ ), [], SIMPLE_PAGE_ORDERING_VERSION );
```

Comment:
```suggestion
					wp_enqueue_style( 'simple-page-ordering', plugins_url( '/dist/css/simple-page-ordering.css', __FILE__ ), [], $script_asset['version'] );
```

WP Scripts calculates the `$script_asset['version']` value based on both the JavaScript and CSS files so the same value can be used for both.

This change deprecates the need for `SIMPLE_PAGE_ORDERING_VERSION` but it will probably need to be kept and updated to account for developers extending this plugin.